### PR TITLE
Fix-issue #1346- added code of conduct linking

### PR DESCRIPTION
**Pull request by ejangphoebe00**
PR Checklist:

- [x] make all edits to the docs in the `src` directory, not in `docs` or in the html files
- [ ] note any issues closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [ ] put any other relevant information below

### CODE REVIEW

**Comment by ejangphoebe00** [comment]
@kathatherine fix for issue #1346

File: src/README.md [hunk 1]
```diff
@@ -9,7 +9,39 @@ To build the docs locally, follow the steps mentioned below:
  6. ``cd ../src``
  7. ``make html``
 
-Documentation update PRs should only change the files in ``src``. 
+Documentation update PRs should only change the files in ``src``.
 
 Once the PR is merged to ``master``, you can find the auto-generated html files in ``docs`` directory.
 
+## conda-forge CODE OF CONDUCT ([https://conda-forge.org/docs/orga/governance.html#code-of-conduct])
```
Comment (kathatherine) [inline src/README.md:16]: The best way to do a link is to pick some text to be the link text and then hide the URL. So this one above could be:

```conda-forge [Code of Conduct](https://conda-forge.org/docs/orga/governance.html#code-of-conduct)```

File: src/README.md [hunk 2]
```diff
@@ -9,7 +9,39 @@ To build the docs locally, follow the steps mentioned below:
...
+
+Conda-forge adheres to the NumFOCUS Code of Conduct.([https://www.numfocus.org/code-of-conduct])
```
Comment (kathatherine) [inline src/README.md:18]: Same thing with this link:

```Conda-forge adheres to the [NumFOCUS Code of Conduct](https://www.numfocus.org/code-of-conduct).```

File: src/README.md [hunk 3]
```diff
@@ -9,7 +9,39 @@ To build the docs locally, follow the steps mentioned below:
...
+                              THE SHORT VERSION
+  >> Be kind to others. Do not insult or put down others. Behave professionally. Remember that harassment and sexist, racist, or exclusionary jokes are not appropriate for NumFOCUS.
+
+  >> All communication should be appropriate for a professional audience including people of many different backgrounds. Sexual language and imagery is not appropriate.
+
+  >> NumFOCUS is dedicated to providing a harassment-free community for everyone, regardless of gender, sexual orientation, gender identity and expression, disability, physical appearance, body size, race, or religion. We do not tolerate harassment of community members in any form.
+
+  Thank you for helping make this a welcoming, friendly community for all.
+
+  The Code of Conduct Reporting Form can be accessed here ([https://numfocus.typeform.com/to/ynjGdT])
```
Comment (kathatherine) [inline src/README.md:29]: And here as well:

```The Code of Conduct Reporting Form can be accessed [here](https://numfocus.typeform.com/to/ynjGdT).```

File: src/README.md [hunk 4]
```diff
@@ -9,7 +9,39 @@ To build the docs locally, follow the steps mentioned below:
...
+
+  Code of Conduct Table of Contents
+
+  >> Diversity Statement([https://numfocus.org/code-of-conduct#diversity-statement])
```
Comment (kathatherine) [inline src/README.md:33]: And all of these links here and below as well.

**Review by kathatherine — CHANGES_REQUESTED** [review]
Thanks! The links need to change.

**Comment by jaimergp** [comment]
Superseded by https://github.com/conda-forge/conda-forge.github.io/pull/1640

**Pull request closed without merging**